extensions/arc/src/constants.ts

```diff
@@ -145,6 +146,10 @@ export class IconPathHelper {
 			light: context.asAbsolutePath('images/gear.svg'),
 			dark: context.asAbsolutePath('images/gear.svg'),
 		};
+		IconPathHelper.gearBlack = {
```

Comment:
Should use a different name for this. Maybe just gear for this and gearBlue for the other one.